opentelemetry-exporters-newrelic/src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicExporters.java

```diff
@@ -55,7 +64,7 @@ public static void start(Configuration configuration) {
             .setExportIntervalMillis(configuration.collectionIntervalSeconds * 1000)
             .setMetricExporter(metricExporterBuilder.build())
             .setMetricProducers(
-                singleton(OpenTelemetrySdk.getGlobalMeterProvider().getMetricProducer()))
+                singleton(((SdkMeterProvider) GlobalMetricsProvider.get()).getMetricProducer()))
```

Comment:
Is this the fix for the metric issue we investigated last week?

Reply:
Unfortunately It doesn't fix the issue that we were troubleshooting last week. However it does replace the deprecated `getGlobalMeterProvider` method.